recipes/opencv/4.x/conanfile.py

```diff
@@ -85,6 +87,8 @@ def config_options(self):
         if self.settings.os != "Linux":
             del self.options.with_gtk
             del self.options.with_v4l
+        if "arm" not in self.settings.arch:
+            del self.options.neon
```

Comment:
Will this work for all the `*arm*`? I mean, we will start to build Macos/M1 binaries and it uses `armv8` as value for the architecture.

Reply:
I'd say time will tell? I personally do not have a Mac, but if its a proper ARMV8, it should have NEON